includes/orders/functions/actions.php

```diff
@@ -362,6 +362,7 @@ function edd_add_manual_order( $args = array() ) {
 				'type_key'    => $type_key,
 				'description' => sanitize_text_field( $adjustment['description'] ),
 				'subtotal'    => floatval( $adjustment['subtotal'] ),
+				'tax'         => floatval( $adjustment['tax'] ),
```

Comment:
`tax` also needs to be added for taxable fees in the `edd_add_order_adjustment` args beginning at line 324. This will fix the issue I'm currently encountering where the tax is not showing in the refund modal for the fee.